pyproject.toml

```
@@ -170,12 +170,7 @@ max-line-length = 130
 max-complexity = 10
 
 [tool.ruff.format]
-exclude = [
-    "versioneer.py",
-    "xdsl/_version.py",
-    "docs/marimo/**/*.py",
-    "docs/marimo/**/*.ipynb",
-]
+exclude = ["docs/marimo/**/*.py", "docs/marimo/**/*.ipynb"]
```

```suggestion
exclude = ["docs/marimo/**/*.py"]
```
?
we don't have any Jupyter notebooks under that path anymore